ci/functional/test_main.sh

```diff
@@ -33,7 +33,7 @@ mkdir -p install/lib/daos/TESTING/ftest/avocado/job-results
 if $TEST_RPMS; then
     # shellcheck disable=SC2029
     ssh -i ci_key -l jenkins "${first_node}" \
-      "TEST_TAG=$test_tag                            \
+      "TEST_TAG=\"$test_tag\"                        \
        TNODES=$tnodes                                \
        FTEST_ARG=$FTEST_ARG                          \
```

Comment:
This should also be quoted.

Reply:
To match the behavior (having launch.py handle any incorrect arguments) of line 46, all three variable assignments should have their values quoted:
```
      "TEST_TAG=\"$test_tag\"                            \
       TNODES=\"$tnodes\"                                \
       FTEST_ARG=\"$FTEST_ARG\"                          \
```

Reply:
This is being done as part of : https://github.com/daos-stack/daos/pull/3160